packages/kbn-es-archiver/src/cli.ts

```diff
@@ -122,6 +122,7 @@ export function runCli() {
       `,
       flags: {
         boolean: ['raw'],
+        string: ['query'],
         help: `
           --raw              don't gzip the archives
```

Comment:
Please describe the `--query` flag here, including that it needs to be properly escaped JSON.